docs/reference_checklist.md

```diff
@@ -37,14 +37,20 @@ After implementing the interpreter:
        );
        ```
 
+1. The author of the PR to provide an explicitly written down testing strategy
+   (in a PR description) similar to
+   [this](https://github.com/openxla/stablehlo/pull/996#issue-1558631158) to use
+   it as a reference while reviewing the verification and type inference methods,
+   and the corresponding tests. The reviewer will make sure that the description
+   is comprehensive.
```

Comment:
Also perhaps replace the "The author of the PR to provide" part with just "Provide" to fit the style of the rest of the document?